packages/ag-charts-community/src/chart/legend.ts

```diff
@@ -726,7 +727,10 @@ export class Legend extends BaseProperties {
     }
 
     private updateItemProxyButtons() {
-        this.itemSelection.each((l) => setElementBBox(l.proxyButton?.listitem, Transformable.toCanvas(l)));
+        this.itemSelection.each((l) => {
+            const bbox = Transformable.toCanvas(l.parent!, this.getLegendItemHitbox(l));
```

Comment:
Sure I'll change it. But in all honesty, I feel like the behaviour of `Transformable.toCanvas` is counter-intuitive. I would have expected `Transformable.toCanvas(l, this.getLegendItemHitbox(l))` to compute the correct bounding box.

Reply:
There is unfortunately still a little nuance to this stuff, mostly because of the need for `Node.getBBox()` to be in the local coordinate space of the parent element to make sense.